### DISCUSSION TITLE
Question set is not appearing in review list after sending it for review

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#382] [Bugs] [answered]
Steps to reproduce

- Login with content creator id
- Create a question set
- Send it for review
- login with content reviewer id
- go to workspace and click on up for review
- check screenshots for more info
![review issue 2](https://user-images.githubusercontent.com/109504832/216600147-da56763b-ed42-4feb-9a8a-201abee0b913.png)
![review issue 1](https://user-images.githubusercontent.com/109504832/216600155-2c12a7d1-e932-4213-b7b6-adcd748d2f02.png)

**Comment by Krishnaj20** [comment]
@sachinb80 - are there any failures in the publish pipeline which is causing this issue? 

@gandham-santhosh @vinukumar-vs

**Reply by Krishnaj20** [reply]
@sachinb80 - Pls do share additional info for the community to check and resolve your query.

@gandham-santhosh @vinukumar-vs @pallakartheekreddy

**Comment by Nadiya496** [comment, marked as answer]
@sachinb80 

We never had the functionality requirement to show the question set for reviewer. Also the question set will be displayed only for the Creator in All my content section alone. 

cc: @pallakartheekreddy

**Reply by Krishnaj20** [reply]
@sachinb80 - this is not a supported feature hence marking this as answered.
